examples/data-objects/smde/src/smde.ts

```diff
@@ -118,10 +93,8 @@ export class SmdeFactory implements IFluidDataStoreFactory {
 				]),
 			),
 			existing,
-			async () => routerP,
+			async (_: IFluidDataStoreRuntime) => SmdeDataObject.load(runtime, existing),
```

Comment:
Is it preferable to capture the runtime from the outer scope here?  Or would it be better to use the runtime passed through the param?